### ISSUE TITLE
Helm chart repo referenced in readme is dead

### ISSUE THREAD

**Issue by rockholla** [istio/istio#10402]
helm charts repo https://storage.googleapis.com/istio-prerelease/daily-build/master-latest-daily/charts as referenced in https://github.com/istio/istio/tree/master/install/kubernetes/helm/istio#installing-the-chart no longer exists.

Looks like maybe it should be updated to https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/master-latest-daily/charts/ per https://github.com/istio/istio.io/pull/2842 ? Btw, I know that some of this is in flux per https://github.com/istio/istio/issues/7505, but it took me quite a bit of digging to find _any_ repo to use that actually worked, so having the current chart repo URL in the README would've helped a lot.

**Comment by blkwtch** [comment]
this one https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/master-latest-daily/charts/ doesn't work either !

**Comment by jmazzitelli** [comment]
Do we even need this anymore? I thought changes were made where the latest Istio builds have all the charts it needs in the distro itself.

See https://github.com/istio/istio/pull/11769 and https://github.com/istio/istio/pull/11767

**Comment by rockholla** [comment]
@jmazzitelli with charts only published in a distro or available via a clone of this repo, I don't see a way of using standard helm requirements/dependencies for pulling from a chart repo/museum, e.g:

```
dependencies:
  - name: istio
    version: 1.0.2
    repository: ???
```

This seems like it could be a common need for others wanting to extend your official chart. Currently I'm using an alternative script in our pipeline to basically recreate what helm does internally for `dep up`, but instead pulling the distro or this repo at a particular version and putting the chart into the `charts/` directory of our chart that wants to extend your istio one.

Anyway, that's the model we're trying to follow, seems like a reasonable and standard helm one, so thought I'd share more if it makes a difference on your side. If you have any ideas of how we might extend your chart outside of the standard helm way and in an easier way than what we're currently doing with custom `dep up` replacement, please let me know. I could be missing something easier here.

**Comment by berstend** [comment]
I had a hard time finding the right combo to get a recent istio working with helm (I didn't see the benefit of checking out the helm chart repo?)

This is what I ended up with, in case anyone else gets as confused as I was 😄:

Makefile / Env:
```makefile
# In what namespace to deploy our own stack
STACK_NAMESPACE ?= default

# --- istio
# Specify istio namespace
ISTIO_NAMESPACE ?= istio-system

# Note: We're using a pretty recent build as some of the issues we've had have only been fixed in 1.1.x 😢
# WARNING: Make sure to test different versions sufficiently before switching production
ISTIO_CHART_REPO ?= https://gcsweb.istio.io/gcs/istio-prerelease/prerelease/1.1.0-snapshot.6/charts/
# - Could also be a specific daily build: https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/release-1.1-20190213-09-16/charts/
# - Could also be the latest daily build: https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/master-latest-daily/charts/
# - Could also be a snapshot build: https://gcsweb.istio.io/gcs/istio-prerelease/prerelease/1.1.0-snapshot.4/charts/
# - List of snapshot releases: https://github.com/istio/istio/releases

# Define the istio image tag for the istio helm chart to use
# WARNING: Make sure to test different versions sufficiently before switching production
# `20190213` is the day `1.1.0-snapshot.6` was released
ISTIO_IMAGE_TAG ?= release-1.1-20190213-09-16
# - Could also be the latest daily build: master-latest-daily
# - List of tags: https://console.cloud.google.com/gcr/images/istio-release/GLOBAL/proxyv2?gcrImageListsize=30
```

Bash:

```bash
# Istio init
kubectl create ns ${ISTIO_NAMESPACE}
helm repo add istio.io ${ISTIO_CHART_REPO}
helm upgrade --install istio-init istio.io/istio-init --namespace ${ISTIO_NAMESPACE}

# Istio install
helm upgrade --install istio istio.io/istio \
  --namespace ${ISTIO_NAMESPACE} \
  --set global.tag=${ISTIO_IMAGE_TAG} \
  --set galley.enabled=true \
  # (...)
  # see: https://github.com/istio/istio/tree/master/install/kubernetes/helm/istio#configuration

kubectl label namespace ${STACK_NAMESPACE} istio-injection=enabled --overwrite
```

Note entirely sure if this is the recommended way to do it, as the docs seem partially outdated on that matter.

**Comment by howardjohn** [comment]
This is no longer an issue with the README - looks like we don't link to any repo anymore and just give instructions for local repo - which is good as this is developer facing

For user facing docs see https://istio.io/docs/setup/kubernetes/install/helm/
